pimd/pim_sock.c

```diff
@@ -77,11 +77,6 @@ void pim_socket_ip_hdr(int fd)
 		if (setsockopt(fd, IPPROTO_IP, IP_HDRINCL, &on, sizeof(on)))
 			zlog_err("%s: Could not turn on IP_HDRINCL option: %m",
 				 __func__);
-#else
-		if (setsockopt(fd, IPPROTO_IPV6, IPV6_HDRINCL, &on, sizeof(on)))
```

Comment:
since the on field here is not used for PIMv6, it will throw compilation error. Please fix.

Reply:
Fixed